Sources/SnapshottingSwift/Snapshots.swift

```diff
@@ -128,6 +128,7 @@ class Snapshots {
       var data = [
         "typeName": preview.typeName,
         "numPreviews": preview.previews.count,
+        "devices": preview.previews.map { $0.device?.rawValue ?? ""}.joined(separator: ",")
```

Comment:
@NicoHinderling do we have to manually do the joined here? This is using JSONSerialization a few lines down so it would be nice if we used that rather than converting the json to a string ourself